script/options.js

```diff
@@ -11,6 +11,7 @@ const defaultHost = 'localhost';
 const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'buildtype', type: String, defaultValue: environments.DEVELOPMENT },
   { name: 'brand-consolidation-enabled', type: Boolean, defaultValue: false },
+  { name: 'metrics-sample-percentage', type: Number, defaultValue: 0 },
```

Comment:
How are you intending this option to get used? Just locally?

Reply:
The intention was for both local use as well as using this option to set sample percentage in `preview.va.gov`

Can we pull from the env to do something like `defaultValue: (ENV['sampleRate'] || 0)`? Or perhaps just decide on a value and hardcode it here (or close to the metrics module)?

Or do you have another suggestion altogether?

Reply:
Typically, if we wanted to vary a value like this, we would have a constant in the code and change it there, rather than a command line option, which requires changing the Jenkinsfile for any change (and restarting the watch task if you're running locally).